components/browser/session/src/main/java/mozilla/components/browser/session/tab/CustomTabConfig.kt

```diff
@@ -88,7 +87,7 @@ class CustomTabConfig internal constructor(
 
             val closeButtonIcon = run {
                 val icon = intent.getParcelableExtra(EXTRA_CLOSE_BUTTON_ICON) as? Bitmap
-                if (icon != null && icon.width <= MAX_CLOSE_BUTTON_SIZE_DP && icon.height <= MAX_CLOSE_BUTTON_SIZE_DP) {
+                if (icon != null) {
```

Comment:
What was the reason for removing the check?

Reply:
This check wasn't exactly working here. The bitmap width was the measured value and we were comparing it to the density pixel value. The close icons provided were actually 24dp but the measured width was 86 so we were always getting the icon as null.

Instead of making the CustomTabConfig depend on Resources/DisplayMetrics to be provided to it, I decided to put that check into the feature itself since we have the context for it to make the internal function `Drawable.isSmallerThan`.

Reply:
That's a good point, I didn't consider that..